<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDlg</class>
 <widget class="QDialog" name="ColorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加背景</string>
  </property>
  <widget class="CButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>35</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/btnReturn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>43</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>设置颜色</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBkgColor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>背景</string>
   </property>
   <widget class="QSlider" name="sliderBkgRed">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelBkgRed">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>红</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderBkgGreen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelBkgGreen">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>绿</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBkgBlue">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>蓝</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderBkgBlue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupFontColor">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>前景</string>
   </property>
   <widget class="QSlider" name="sliderFontRed">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelFontRed">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>红</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderFontGreen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelFontGreen">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>绿</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFontBlue">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>蓝</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderFontBlue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CButton</class>
   <extends>QPushButton</extends>
   <header location="global">button.h</header>
  </customwidget>
  <customwidget>
   <class>CLabel</class>
   <extends>QLabel</extends>
   <header location="global">label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
